### PR TITLE
[PM-5384] Create Duo Redirect Connector

### PR DESCRIPTION
**Pull request by rr-bw**
## Type of change

- New feature development

## Objective

Creates a Duo redirect connector

## Code changes

- `apps/web/src/connectors/duo-redirect.ts` - add duo-redirect to handle browser/web and mobile/desktop
- `apps/web/src/connectors/duo-redirect.html` - add HTML
- `apps/web/webpack.config.js` - add to webpack

## Screencast

https://github.com/bitwarden/clients/assets/102181210/b6488629-592f-43e8-82a5-24153cda8604

### CODE REVIEW

File: apps/web/src/connectors/duo-redirect.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width"
+    />
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src 'self'; child-src 'self' https://*.duosecurity.com https://*.duofederal.com; frame-src 'self' https://*.duosecurity.com https://*.duofederal.com;"
+    />
```
Comment (jlf0dev) [inline apps/web/src/connectors/duo-redirect.html:12]: I know I copied this over, but we probably don't need a CSP

File: apps/web/src/connectors/duo-redirect.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <title>Bitwarden Duo Redirect Connector</title>
+  </head>
+
+  <body></body>
```
Comment (jlf0dev) [inline apps/web/src/connectors/duo-redirect.html:16]: Can you look into showing a message such as, "You may close this window and continue in <client>" similar to how we do for SSO? It's just a cookie named `ssoHandOffMessage` that we put on the window.

Comment (jlf0dev) [reply apps/web/src/connectors/duo-redirect.html:16]: We can't hardcode it because we need translation

Comment (JaredSnider-Bitwarden) [reply apps/web/src/connectors/duo-redirect.html:16]: Do we also need a loading spinner for good UX here?

**Comment by rr-bw** [comment]
@jlf0dev @JaredSnider-Bitwarden I uploaded a screencast for clarify

**Review by jlf0dev — APPROVED** [review]
Great work!

**Pull request merged**